specification/common-types/resource-management/v6/mobo.json

```diff
@@ -0,0 +1,38 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Common types",
+    "version": "6.0"
+  },
+  "paths": {},
+  "definitions": {
+    "ManagedOnBehalfOfConfiguration": {
+      "type": "object",
+      "description": "Managed-On-Behalf-Of configuration properties. This configuration exists for the resources where a resource provider manages those resources on behalf of the resource owner.",
+      "properties": {
+        "moboBrokerResources": {
+          "type": "array",
+          "description": "Managed-On-Behalf-Of broker resources",
+          "items": {
+            "$ref": "#/definitions/MoboBrokerResource"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": [
+            "id"
+          ]
+        }
+      }
+    },
+    "MoboBrokerResource": {
+      "type": "object",
+      "description": "Managed-On-Behalf-Of broker resource. This resource is created by the Resource Provider to manage some resources on behalf of the user.",
```

Comment:
> Resource Provider [](http://example.com/codeflow?start=91&length=18)

nit, don't need to capitalize resource provider

Reply:
this is capitalized in v5: https://github.com/Azure/azure-rest-api-specs/blob/0bbf10474eb460cb35d7e2d3524054418eb19406/specification/common-types/resource-management/v5/mobo.json#L28 

SDK team is not really onboard with us changing the description, so just double checking with you that this is needed?

Reply:
It is just suggesting style cleanup - in line with usual MS doc style guidelines.